### PR TITLE
[Doc] Edit for readability

### PR DESCRIPTION
**Pull request by john-andrilla**
Edit pass for grammar and style. 
* A great value-add would be to provide assumptions and prerequisites in the opening section. This helps readers understand what they need to have in place in order to make use of your tutorial steps.
* The wikidata knowledge graph could be improved with a smaller font for the Zuckerberg circle.

## Description
Edit pass for grammar and style

### CODE REVIEW

File: tutorials/hetero/1_basics.py [hunk 1]
```diff
@@ -191,26 +188,26 @@
 print(pa_g.successors(1, etype='written-by'))  # get the authors that write paper #1
 
 # Type name argument could be omitted whenever the behavior is unambiguous.
-print(pa_g.number_of_edges())  # only one edge type, the edge type argument could be omitted
+print(pa_g.number_of_edges())  # Oly one edge type, the edge type argument could be omitted
```
Comment (jermainewang) [inline tutorials/hetero/1_basics.py]: typo here

Comment (john-andrilla) [inline tutorials/hetero/1_basics.py]:
```suggestion
print(pa_g.number_of_edges())  # Only one edge type, the edge type argument could be omitted
```

Comment (john-andrilla) [reply tutorials/hetero/1_basics.py]: Thanks. Committed this update.

File: tutorials/hetero/1_basics.py [hunk 2]
```diff
@@ -156,33 +153,33 @@
...
 # The dataset stores node information by their types: ``P`` for paper, ``A``
-# for author, ``C`` for conference, ``L`` for subject code, etc. The relations
-# are stored as scipy sparse matrix under key ``XvsY``, where ``X`` and ``Y``
-# could be any of the node type codes.
+# for author, ``C`` for conference, ``L`` for subject code, and so on. The relationships
+# are stored as SciPy sparse matrix under key ``XvsY``, where ``X`` and ``Y``
+# could be any of the node type code.
 #
-# The following codes print out some statistics about the paper-author relation.
+# The following code prints out some statistics about the paper-author relations.
```
Comment (aaronmarkham) [inline tutorials/hetero/1_basics.py]:
```suggestion
# The following code prints out some statistics about the paper-author relationships.
```

File: tutorials/hetero/1_basics.py [hunk 3]
```diff
@@ -108,41 +106,40 @@
...
+# Creating from networkx graph
 import networkx as nx
 plus1 = nx.DiGraph()
 plus1.add_nodes_from(['u0', 'u1', 'u2'], bipartite=0)
 plus1.add_nodes_from(['m0', 'm1'], bipartite=1)
 plus1.add_edges_from([('u0', 'm0'), ('u0', 'm1'), ('u1', 'm0')])
-# To simplify the example, we reuse the minus1 object.
+# To simplify the example, reuse the minus1 object.
 # This also means that you could use different sources of graph data
 # for different relations.
```
Comment (aaronmarkham) [inline tutorials/hetero/1_basics.py]:
```suggestion
# for different relationships.
```

File: tutorials/hetero/1_basics.py [hunk 4]
```diff
@@ -191,26 +188,26 @@
...
+# All the ntype and etype arguments could be omitted because the behavior is unambiguous.
 print(pp_g.number_of_nodes())
 print(pp_g.number_of_edges())
 print(pp_g.successors(3))
 
 ###############################################################################
-# We then create a subset of the ACM graph using the paper-author, paper-paper
-# and paper-subject relations.  Meanwhile, we should also add the reverse
+# Create a subset of the ACM graph using the paper-author, paper-paper, 
+# and paper-subject relations.  Meanwhile, also add the reverse
```
Comment (aaronmarkham) [inline tutorials/hetero/1_basics.py]:
```suggestion
# and paper-subject relationships.  Meanwhile, also add the reverse
```

File: tutorials/hetero/1_basics.py [hunk 5]
```diff
@@ -191,26 +188,26 @@
...
 # relations to prepare for the later sections.
```
Comment (aaronmarkham) [inline tutorials/hetero/1_basics.py]:
```suggestion
# relationship to prepare for the later sections.
```

File: tutorials/hetero/1_basics.py [hunk 6]
```diff
@@ -297,27 +294,27 @@ def plot_graph(nxg):
...
-# Breaking down the equation, we see that there are two parts in the
-# computation:
+# Breaking down the equation, you see that there are two parts in the
+# computation.
 #
-# (i) message computation and aggregation within each relation :math:`r`, and
+# (i) Message computation and aggregation within each relation :math:`r`
 #
-# (ii) reduction that merges the results from multiple relations.
+# (ii) Reduction that merges the results from multiple relations
```
Comment (aaronmarkham) [inline tutorials/hetero/1_basics.py]:
```suggestion
# (ii) Reduction that merges the results from multiple relationships
```

**Pull request merged**